docs/tempo/website/architecture/_index.md

```diff
@@ -5,5 +5,59 @@ weight: 500
 
 A collection of documents that detail Tempo architectural decisions and operational implications.
 
-- [Architecture](architecture/)
-- [Consistent Hash Ring](consistent-hash-ring/)
+# Architecture
+
+This document provides an overview of the major components that comprise Tempo.  Please refer to [the examples](https://github.com/grafana/tempo/tree/main/example) to see some deployment options.
+
+<p align="center"><img src="tempo_arch.png" alt="Tempo Architecture"></p>
+
+## Tempo
+
+### Distributor
+
+Accepts spans in multiple formats including Jaeger, OpenTelemetry, Zipkin.
+Routes spans to ingesters by hashing the `traceID` and using a [distributed consistent hash ring]({{< relref "consistent-hash-ring" >}}).
+
+The distributor uses the receiver layer from the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
+For best performance it is recommended to ingest [OTel Proto](https://github.com/open-telemetry/opentelemetry-proto).  For this reason
```

Comment:
```suggestion
For best performance, ingest [OTel Proto](https://github.com/open-telemetry/opentelemetry-proto).  For this reason
```

Reply:
Recommended seems right here. Also makes sense in the following sentence. Feel free to reopen if you feel otherwise!